emulators.tex

```diff
@@ -8,14 +8,127 @@ \chapter{Emulators}
+
+When using a real Mega 65 a keyboard layout related to the old Commodore machines is
+encountered. How can this be adapted to a PC/Mac ? It's not that hard. Most of the keys
+for writing stay the same - given an American layout. So also special characters reached
+by \specialkey{Shift} and a number are same to a U.S. keyboard.
+
+The closer you get to \specialkey{Enter} or \widekey{Backspace} keys you will find
+a whole column of keys missing. Yes, even a C64-keyboard had this extra set of keys. For
+Xemu those are mainly operated with the six keys above the cursor keys. There you also
+find \specialkey{RUN STOP}, completely off in comparison to C64 emulator like VICE. The reason is, a Mega 65 even has an \specialkey{Esc} key. So some keys are approaching a
+more modern layout.
+
+\begin{center}
+  \includegraphics[width=\linewidth]{images/xemu-extended-keyboard.png}
+\end{center}
+
+As you might spot quickly with this setup \specialkey{RUN STOP} \widekey{RESTORE} could
+be done single-handed.
+
+Please be aware this image shows graphics symbols being reached in the orientation the
+\megasymbolkey and \specialkey{Shift} key are positioned on a real Mega 65. On a PC
+keyboard it is crossovered, unfortunately. So, while \megasymbolkey \megakey{S} gives
+you the upper right corner of a rectangle \specialkey{Shift} \megakey{S} gives you the
+heart symbol every new C64 owner must have tried first in the eightees.
+
+For the PC styled special character keys the lower symbol is reached without the use
+of modifier keys. The upper symbol shows what you get in conjunction with the
+\specialkey{Shift} key.
+
+Xemu lets you have two joysticks (one at a time) on the numeric keypad.
+\specialkey{Enter} over there is used to toggle ports 1, 2 and to off when pressed
+repeatedly.
+
+Please be aware the emulator by default catches \megakey{F9} \megakey{F10}
+\megakey{F11} as shortcuts of its own functions. If you want those keys to be used by
+the emulated Mega 65 you need to setup an own keyboard mapping. The files are found
+in the folder for Xemu's settings as explained on page
+\pageref{sec:sdcard-settings-location}. This is done by copying
+\stw{keymap-default.cfg} into \stw{keymap.cfg} and then removing the following lines:
+
+\begin{tcolorbox}[colback=black,coltext=white]
+\verbatimfont{\codefont}
+\begin{verbatim}
+XEMU-EXIT F9
+XEMU-FULLSCREEN F11
+\end{verbatim}
+\end{tcolorbox}
+
+and changing:
+
+\begin{tcolorbox}[colback=black,coltext=white]
+\verbatimfont{\codefont}
+\begin{verbatim}
+F9 Unknown
+F11 Unknown
+\end{verbatim}
+\end{tcolorbox}
+
+into:
+
+\begin{tcolorbox}[colback=black,coltext=white]
+\verbatimfont{\codefont}
+\begin{verbatim}
+F9 F9
+F11 F11
+\end{verbatim}
+\end{tcolorbox}
+
+Fullscreen and exiting Xemu is still available in the menu you reach by right mouse
+clicking into the emu's main window.
+
+\subsection{Apple MacOS start problems}
+
+If a message \stw{macOS cannot verify that this app is free from malware} pops up you
```

Comment:
"you may want to control-click to start it:" -> "use the following procedure when you open the app for the first time:"

Reply:
You're right. There I had "control-click" twice. Made no sense. Yours is now in.